packages/storybook/src/docs/nimble-intro.mdx

```diff
@@ -10,13 +10,15 @@ import nimbleLogo from './nimble-logo-icon.svg';
 
 ## Overview
 
-The [Nimble Design System](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/) is an easy to use collection of well-designed,
+The Nimble Design System is an easy to use collection of well-designed,
```

Comment:
kinda just want the first real link to be nimble related and night spright related.

```suggestion
The [Nimble Design System](https://nimble.ni.dev/) is an easy to use collection of well-designed,
```